benchexec/tools/sikraken.py

```diff
@@ -0,0 +1,53 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+This class serves as BenchExec tool adaptor for Sikraken for Test-Comp
+See baseclass documentation at https://github.com/sosy-lab/benchexec/blob/main/benchexec/tools/template.py
+"""
```

Comment:
The comment here and below is mostly redundant. One or both could be removed.

Reply:
yes sorry.